packages/apt/install_to_ubuntu.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -exu
+
+package=$1
+
+sudo apt update
+sudo apt-get install -y -V software-properties-common lsb-release
```

Comment:
Could you use `apt` instead of `apt-get`?

Reply:
Ah...
I've fixed it.